subprojects/core/build.gradle.kts

```diff
@@ -180,8 +181,8 @@ dependencies {
     testRuntimeOnly(project(":distributions-core")) {
         because("ProjectBuilder tests load services from a Gradle distribution.")
     }
-    integTestDistributionRuntimeOnly(project(":distributions-basics")) {
-        because("Some tests utilise the 'java-gradle-plugin' and with that TestKit")
+    integTestDistributionRuntimeOnly(project(":distributions-jvm")) {
+        because("Some tests utilise the 'java-gradle-plugin' and with that TestKit, some also use the 'war' plugin")
```

Comment:
The `:core` integ tests use a lot more than just the war plugin. IMO we should just say something like `also use JVM plugins`

Reply:
I'll get this with the next project, for now, the War Plugin is the only new thing it needs here.